### PR TITLE
Italian translation update strings and style

### PR DESCRIPTION
**Pull request by albertopasqualetto**
Italian translation update strings and style as english locale

### CODE REVIEW

File: web/public/locales/it.json [hunk 1]
```diff
@@ -195,407 +235,1351 @@
...
+      "zoneName": "Isole Faroe"
+    },
+    "FO-MI": {
+      "countryName": "Faroe Islands",
+      "zoneName": "Isole Principali"
+    },
+    "FO-SI": {
+      "countryName": "Faroe Islands",
+      "zoneName": "Suðuroy"
+    },
```
Comment (VIKTORVAV99) [inline web/public/locales/it.json:652]: Shouldn't the `countryName` here be Isole Faroe?

Comment (albertopasqualetto) [reply web/public/locales/it.json:652]: Yes, you are right, I am going to commit it now

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Comment by VIKTORVAV99** [comment]
Can you run prettier so this passes formatting? Beyond that it looks good to me.

**Pull request merged**